lib/Request.js

```diff
@@ -121,13 +121,31 @@ Request.prototype.getQueries = function() {
 
 Request.prototype.setQuery = function(query, value) {
     this._url.query[query] = value;
+
     return this;
 };
 
 Request.prototype.getQuery = function(query) {
-    return this._url.query.hasOwnProperty(query) ? this._url.query[query] : undefined;
+    return getQueryValue(this._url.query, query);
 };
 
+function getQueryValue(queryObject, query) {
```

Comment:
i'll have to modify this._url.query to have it in the getQuery function as part of the recursion... i do not want to do that